android/envoy/src/main/kotlin/org/greatfire/envoy/NetworkIntentService.kt

```diff
@@ -283,6 +291,10 @@ class NetworkIntentService : IntentService("NetworkIntentService") {
                 broadcastValidationFailure()
             }
         } else {
+
+            // set aside certain urls to test first
+            var preferredUrls = Collections.synchronizedList(mutableListOf<String>())
```

Comment:
why is this synchronized?  i don't see this being edited from more than one thread?

Reply:
oops, i was originally declaring it as a class variable so it had the same type as the other lists.